js/components/AccountInput/AccountInput.js

```diff
@@ -2,12 +2,15 @@ import React from 'react';
 import { TextInput } from 'react-native';
 import styles from './styles';
 
-export const AccountInput = ({ input, placeholder, password }) => {
+export const AccountInput = ({ input, placeholder, password, invalid }) => {
   return (
     <TextInput
-      style={styles.input}
+      autoCapitalize="none"
+      style={
+        invalid ? [styles.input, styles.invalid] : [styles.input, styles.valid]
+      }
       placeholder={placeholder}
-      placeholderTextColor="#0082B580"
+      placeholderTextColor={invalid ? '#d84934' : '#0082B580'}
```

Comment:
hash values are supposed to be 6 chars only. Please change it.

Reply:
The design specifies a 50% transparency on the main brand blue and react-native docs said rgba hex were valid for transparency. Is there a different preferred way to do this?